test/terra/reference/ref_kraus_noise.py

```diff
@@ -60,15 +60,6 @@ def kraus_gate_error_noise_models():
 
     return noise_models
 
-def kraus_gate_error_noise_models_full():
-    """Kraus gate error noise models on many gate types"""
-
-    # Amplitude damping error on "u1", "u2", "u3", "cx"
-    error = amplitude_damping_error(0.2)
-    noise_model = NoiseModel()
-    noise_model.add_all_qubit_quantum_error(error, ['h'])
-    noise_model.add_all_qubit_quantum_error(error.tensor(error), ['cp', 'swap'])
-    return noise_model
 
```

Comment:
Was this function completely unused?

Reply:
It was used in a single test so I just moved it to make it easier to read

Reply:
Oh yeah, I totally misread the test you modified - I only noticed the other helper, not this one.